jrnl 2.2.0

elliotmassen opened the pull request:
This PR contains the updated `jrnl` formula for version 2.2.1. It actually uses [2.2.1b2 from PyPi](https://pypi.org/project/jrnl/2.2.1b2). Perhaps this will need to be bumped to 2.2.2 so as not to avoid confusion?

I've been following [this guide](https://docs.brew.sh/How-To-Open-a-Homebrew-Pull-Request#create-your-pull-request-from-a-new-branch) and am for some reason unable to run ` brew test jrnl`, I'd really appriciate it if someone can give it a go and let me know if it works for you.

After considering the version and the testing then we can open a PR on [homebrew/homebrew-core](https://github.com/Homebrew/homebrew-core).

wren commented:
Hi, I would expect the current tests to fail since we don't store the config files in the same location. Is that what you're saying? Or you can't run it at all?

elliotmassen commented:
@wren I wasn't sure if you were intending to use the new tap beta version now or whether that was for your automated solution, so I've updated this just in case. I've used the asset you suggested in the issue and also I updated the test as you did in the new tap repo. Give it a go, the testing now works - if you're happy with it, I'm still happy to deal with opening a PR on homebrew-core. No worries if not :)

wren commented:
The beta is only there to test all the hard work you did. Once we test it a little more, I'd love to have you submit it to the Homebrew core. Thank you, again!
